webapp/components/Comment.vue

```diff
@@ -10,7 +10,7 @@
     </ds-card>
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
-    <ds-card>
+    <ds-card :id="`commentId-${comment.id}`">
```

Comment:
@mattwr18 and I did it for the user experience. That a user would have a chance for a clue about what this is, even in case the scroll might not working. @alina-beck